server/cli/launch.py

```diff
@@ -148,7 +153,16 @@ def launch(
         log = logging.getLogger("werkzeug")
         log.setLevel(logging.ERROR)
 
-    click.echo(f"[cellxgene] Loading data from {basename(data)}, this may take awhile...")
+    file_size = getsize(data) if exists(data) else 0
```

Comment:
Do we need this exists check since click already checks that for us? (Line 19) I'm ok with leaving it in though.

Reply:
I suppose not.  I'm just paranoid.  Will remove.